src/blocks.tsx

```diff
@@ -0,0 +1,343 @@
+import * as React from 'react';
```

Comment:
please, move all these components to the community folder.
FYI: https://github.com/Telefonica/mistica-web#mistica-community and https://mistica-web.vercel.app/?path=/story/community-welcome--default